src/lib/util.js

```diff
@@ -50,4 +51,4 @@ module.exports = {
     htmlEncode: function (value) {
         return !(value == null) ? value.replace(/&/g, '&amp;').replace(/"/g, '&quot;').replace(/'/g, '&#39;').replace(/</g, '&lt;').replace(/>/g, '&gt;') : '';
```

Comment:
Thats fine... not sure why it was done that way. My guess is that there was more logic originally.